docs/releases/5.2.txt

```diff
@@ -308,6 +308,9 @@ Models
 * :meth:`.QuerySet.explain` now supports the ``memory`` and ``serialize``
   options on PostgreSQL 17+.
 
+* Added model check for a clash between model field's ``related_name`` and
+  model's manager name.
+
```

Comment:
This can be removed, we typically don't mention new system checks in release notes.

Reply:
Thanks @felixxm :) I'll remove it.